build.gradle

```diff
@@ -92,10 +93,9 @@ subprojects {
             }
         }
     }
-}
 
 def excludeProjects() {
-    subprojects.findAll { project -> (project.name != 'proto' && project.name != 'osgi-wrappers')}
+    project.name != 'proto' && project.name != 'osgi-wrappers'
```

Comment:
Yes, this condition is not at all valid as its a single module project, just 1 project "Grabbit" was returned, which is root project now, so removing this configuration block.